presto-spi/src/test/java/com/facebook/presto/spi/type/TestTypeSignature.java

```diff
@@ -151,7 +159,7 @@ private static void assertSignature(
         for (int i = 0; i < signature.getParameters().size(); i++) {
             assertEquals(signature.getParameters().get(i).toString(), parameters.get(i));
         }
-        assertEquals(signature.toString(), expectedTypeName);
+        assertEqualsIgnoreCase(signature.toString(), expectedTypeName);
```

Comment:
I don't expect to support that for a while (it requires changes all over the place), but good point. Let me see if I can make it more conservative.